engine/link/mystery_gift.asm

```diff
@@ -1186,15 +1186,15 @@ ReceiveIRDataMessage:
 	inc d
 	jr z, .recv_done
 	ldh a, [c]
-	bit rRP_RECEIVING, a
+	bit 1, a ; RPF_WRITE_HI in hardware.inc, RPB_DATAIN needs to be added for this
 	jr z, .recv_loop
 	ld d, 0
 .recv_done
 .send_loop
 	inc d
 	jr z, .send_done
 	ldh a, [c]
-	bit rRP_RECEIVING, a
+	bit 1, a ; RPF_WRITE_HI in hardware.inc, RPB_DATAIN needs to be added for this
```

Comment:
Then you should open a PR or at least an issue there, otherwise this will be forgotten about.